profiles/common/modules/features/multisite_wysiwyg/multisite_wysiwyg.behat.inc

```diff
@@ -279,4 +285,106 @@ class MultisiteWysiwygSubContext extends RawDrupalContext implements DrupalSubCo
     return $label_value;
   }
 
+  /**
+   * Switch frames inside the media dialogs.
+   *
+   * @When I switch to the frame :frame
+   */
+  public function iSwitchToTheFrame($frame) {
+    $this->getSession()->switchToIFrame($frame);
+    $this->iframe = $frame;
+    $this->getSession()->wait(5000);
```

Comment:
Yes, unfortunately if we do not wait the frame is not loaded soon enough before our test continues, and bits of code are missing.